src/main/java/com/recycle/buddy/model/Image.java

```diff
@@ -0,0 +1,23 @@
+package com.recycle.buddy.model;
+
+public class Image {
+
+    private String id;
+    private String base64;
+
+    public String getId() {
```

Comment:
If we decide to make the the class immutable, eliminate the setter methods. Either way, I feel that assigning values to the data fields can be done at object creation, so a two arg constructor should probably be provided.

Reply:
Jackson framework that binds JSON data with object  requires JavaBean style class.
 I got this message, when tried to use a constructor:
 "Cannot construct instance of `com.recycle.buddy.model.Image` (no Creators, like default construct, exist): cannot deserialize from Object value (no delegate- or property-based Creator)"